packages/cubejs-server-core/core/FileRepository.js

```diff
@@ -11,13 +11,25 @@ class FileRepository {
     return path.join(process.cwd(), this.repositoryPath);
   }
 
+  async getFiles(dir, fileList = []) {
+    const files = await fs.readdir(dir);
+    for (const file of files) {
+      const stat = await fs.stat(path.join(dir, file));
```

Comment:
Here should be `path.join(this.localPath(), dir, file)` instead of `path.join(dir, file)`.